src/python/native_common/ext_modules.py.in

```diff
@@ -22,6 +22,29 @@ import sys
 
 from setuptools import Extension
 
+class BadAutoconfSubstitutes(ValueError):
+    pass
+
+def _get_ac_subst_bool(true_subst, false_subst):
+    """
+        This function accepts arguments which are supposedly a result
+        of substitution by the code in the configure script that
+        was generated for a boolean flag.
+        It either returns the value of this flag, or fails if the
+        combination of these "substitution results" is not valid.
```

Comment:
If you keep this function, can you add an example to the comment? I had to read through the usages below before I understood this function.

Reply:
Added an example. Hope that this will be helpful for the readers.